content/english/java-datastructures/2d-arrays/operations.md

```diff
@@ -29,8 +29,14 @@ public static void main(String[] args)
 { 
```

Comment:
Can you fix this line to be consistent with how we use brackets?

```
public static void main(String[] args) { 
```

Reply:
both from this page should be resolved now.